avocado/utils/multipath.py

```diff
@@ -106,6 +106,37 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "multipathd show maps format '%n %w'"
+    wwids = process.run(cmd, ignore_status=True,
```

Comment:
This should be inside a try/except? The command could be missing on the host for instance, or a permission denied could be raised.

Reply:
Process.run has the error handling and reporting mechanism. Hence I think a separate try/except it not required here

Reply:
@harihare if `process.run()` raises exceptions we should catch them on the caller and raise a more high-level exception. Otherwise, the user will receive a low-level exception and not a specif one. In this case a try...except would be useful here to raise another exception saying that the getting the wwid binding for a given mpath was failed plus the low-level message that `process.run()` sent to us.

Reply:
@beraldoleal Since the whole multipath.py would need to handle exceptions in most of the functions, can we pursue that in a separate patch ?

Reply:
@narasimhan-v the whole multipath.py file would be nice for the future, sure. But since that his PR is adding a few methods I'm requesting to add the exceptions only on the methods added here.